include/ck/tensor_operation/gpu/device/device_reduce_blockwise.hpp

```diff
@@ -148,26 +154,27 @@ struct DeviceReduceBlockWise : public DeviceReduce<InElementwiseOperation, AccEl
         {
             (void)workspace_dev;
 
-            inLengths_  = inLengths;
-            inStrides_  = inStrides;
             outLengths_ = outLengths;
             outStrides_ = outStrides;
 
+            std::tie(inLengths_, inStrides_) =
+                shuffle_tensor_dimensions<Rank, NumReduceDims>(inLengths, inStrides, toReduceDims);
+
             in_elementwise_op_  = in_elementwise_op;
```

Comment:
Move this into member initializer list

Reply:
Do you mean move `outLengths_ = outLengths; outStrides_ = outStrides;` and `in_elementwise_op_  = in_elementwise_op; acc_elementwise_op_ = acc_elementwise_op` to the member initializer list ?